people/admin.py

```diff
@@ -9,14 +9,14 @@ class PersonAdmin(admin.ModelAdmin):
     Admin for Person model
     """
 
```

Comment:
You could use `list_editable` which would allow the client to update the `order` in the change list view instead of going into each record.

Reply:
Good idea. They'll have to bulk update a few when this is pushed so will add it in now